js/manager.js

```diff
@@ -247,19 +247,27 @@ const DFPManager = Object.assign(new EventEmitter().setMaxListeners(0), {
     }, slots);
   },
 
-  refresh() {
+  refresh(...slots) {
+
+    const slotsToRefresh = (slots) => {
+      const refreshableSlots = this.getRefreshableSlots();
+      if (slots.length === 0) {
+        return Object.keys(refreshableSlots).map(slotId => refreshableSlots[slotId].gptSlot);
+      }
+      return slots.map(slotId => refreshableSlots[slotId].gptSlot);
```

Comment:
I think in this case we'll expect to refresh any arbitrary slot, just ignoring any configuration passed via `shouldRefresh`, so in this line I believe we should use `registeredSlots`, this will make things less confusing, what do you think?
```suggestion
      return slots.map(slotId => registeredSlots[slotId].gptSlot);
```

Reply:
agreed, have changed 👍